Dockerfile

```diff
@@ -41,7 +41,8 @@ RUN foundryup
 
 RUN git clone https://github.com/ethereum-optimism/optimism.git && \
     cd optimism && \
-    git checkout tutorials/chain && \
+    git checkout develop && \
+    git pull origin develop && \
```

Comment:
prolly worth using something like `git clone --depth 1 https://path/to/repo/foo.git -b bar`? so we do a shallow clone of just 1 branch?

Reply:
use the suggestions, so I can just merge with one click

Reply:
meh, v2 work